lib/Sema/TypeCheckDeclObjC.cpp

```diff
@@ -2032,6 +2034,7 @@ std::pair<unsigned, DeclName> swift::getObjCMethodDiagInfo(
       llvm_unreachable("Not an Objective-C entry point");
 
     case AccessorKind::Get:
+    case AccessorKind::DistributedGet:
```

Comment:
I think this should hit `Not an Objective-C entry point` for distributed getter.